packages/synapse-react-client/src/components/QueryVisualizationWrapper/QueryVisualizationContext.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { createContext, useContext } from 'react'
+import { ColumnType } from '@sage-bionetworks/synapse-types'
+
+export type QueryVisualizationContextType = {
```

Comment:
Just separated `QueryVisualizationContext` into its own file, which lets Vite HMR work on this file, no changes